modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+
+
+def get_iou(new, gt):
+    '''
+    During the calculation of intersection over union, we are checking
+    numerical value of area_of_overlap, because if it is equal to 0,
+    we have no intersection.
+    '''
+    new_xmin = new[0]
+    new_xmax = new[0] + new[2] - 1.0
+    new_ymin = new[1]
+    new_ymax = new[1] + new[3] - 1.0
+    gt_xmin = gt[0]
+    gt_xmax = gt[0] + gt[2] - 1.0
+    gt_ymin = gt[1]
+    gt_ymax = gt[1] + gt[3] - 1.0
+    dx = max(0, min(new_xmax, gt_xmax) - max(new_xmin, gt_xmin))
+    dy = max(0, min(new_ymax, gt_ymax) - max(new_ymin, gt_ymin))
+    area_of_overlap = dx * dy
+    area_of_union = (new_xmax - new_xmin) * (new_ymax - new_ymin) + (
+        gt_xmax - gt_xmin) * (gt_ymax - gt_ymin) - area_of_overlap
+    if area_of_overlap != 0 and area_of_union != 0:
```

Comment:
Why are you adding `area_of_overlap != 0`?
`iou = area_of_overlap / area_of_union if area_of_union != 0 else 0`
better for me